src/Parser/DeprecatedHelperVisitor.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace mglaman\PHPStanDrupal\Parser;
+
+use PhpParser\Node;
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitor;
+use PhpParser\NodeVisitorAbstract;
+
+class DeprecatedHelperVisitor extends NodeVisitorAbstract implements NodeVisitor
+{
+
+    private ?Node\Arg $deprecatedCall = null;
+
+    public function enterNode(Node $node)
+    {
+        if ($node instanceof Node\Expr\StaticCall) {
+            if ($node->class instanceof Node\Name\FullyQualified && $node->class->toString() === 'Drupal\Component\Utility\DeprecationHelper') {
+                $this->deprecatedCall = $node->getArgs()[2];
+                return null;
+            }
+        }
+
+        if ($this->deprecatedCall !== null && $node instanceof Node\Arg && $this->isSavedArgument($node)) {
+            $this->deprecatedCall = null;
+            return NodeTraverser::DONT_TRAVERSE_CURRENT_AND_CHILDREN;
+        }
+
+        return null;
+    }
```

Comment:
Is there a chance our property won't get unset for some reason?

Reply:
Calling deprecation helper without arguments? But that is impossible. 

Perhaps if while moving into the function something changes the node, so the argument is no longer on the same line and therefor not equal? Not sure how, and if that is possible. 

But the effect would basically be if would check the array a few times extra, which should be quite cheap.

There might be a way to add some more checks when in the argument. Or do something like keeping a stack of the last static call and check if that call is nothing weird. 

```
        if ($node instanceof Node\Expr\StaticCall) {
            $this->staticCall = $node; // this and check this?
            if ($node->class instanceof Node\Name\FullyQualified && $node->class->toString() === 'Drupal\Component\Utility\DeprecationHelper') {
                $this->deprecatedCall = $node->getArgs()[2];
                return null;
            }
        }
```

But that seems like overkill.